### PR TITLE
Feature/added twig linter

### PR DESCRIPTION
**Pull request by newmedia**
Add new twig linter for twig version 3.x
Added 1.x version remark to other twig linter

### CODE REVIEW

**Comment by jakzal** [comment]
Thanks for this @newmedia. Can you rebase with latest master please?

File: resources/linting.json [hunk 1]
```diff
@@ -52,6 +52,20 @@
...
             "tags": ["linting"]
+        },
+        {
+            "name": "twig-linter",
+            "summary": "Standalone cli twig 3.X linter",
+            "website": "https://github.com/sserbin/twig-linter",
+            "command": {
+                "composer-bin-plugin": {
+                    "package": "sserbin/twig-linter:@dev",
+                    "namespace": "tools",
```
Comment (jakzal) [inline resources/linting.json:63]: To avoid conflicts with other tools, can we change `tools` to `twig-linter` please?

Comment (newmedia) [reply resources/linting.json]: @jakzal done

**Comment by newmedia** [comment]
when i rebase the branch says i'm up to date! @jakzal

**Comment by jakzal** [comment]
@newmedia thanks. 

The build failure might be related to https://github.com/box-project/box/issues/586.

**Comment by newmedia** [comment]
mm it also looked like the rebase wasn't doing what i expected.. retried with clean checkout and now i have updates.. Pushed to the pull request again. lets see if this builda 👍🏽

**Comment by newmedia** [comment]
Any updates on the box issue?

**Comment by jakzal** [comment]
Thank you for you contribution @newmedia 🍺 

Merged as #417 as git history was messed up here.

**Pull request closed without merging**